lib/modules/apostrophe-schemas/public/js/user.js

```diff
@@ -1513,13 +1513,14 @@ apos.define('apostrophe-schemas', {
     self.addFieldType({
       name: 'color',
       populate: function (data, name, $field, $el, field, callback) {
-        $field.val(data[name]);
-        apos.ui.enhanceColorpicker($field);
+        var options = field.spectrumOptions || {};
+        apos.ui.enhanceColorpicker($field, data[name], options);
```

Comment:
... Looks like options was never passed before, so I guess there's no real bc break in reshuffling to put a second argument in the middle 👍